pkg/server/application_api/storage_inspection_test.go

```diff
@@ -336,50 +336,6 @@ func BenchmarkAdminAPIDataDistribution(b *testing.B) {
 	b.StopTimer()
 }
 
-func TestHotRangesResponse(t *testing.T) {
-	defer leaktest.AfterTest(t)()
-	defer log.Scope(t).Close(t)
-
-	defer serverutils.TestingSetDefaultTenantSelectionOverride(
-		// bug: HotRanges not available with secondary tenants yet.
-		base.TestIsForStuffThatShouldWorkWithSecondaryTenantsButDoesntYet(109499),
-	)()
-
-	srv := rangetestutils.StartServer(t)
-	defer srv.Stopper().Stop(context.Background())
-	ts := srv.ApplicationLayer()
-
-	var hotRangesResp serverpb.HotRangesResponseV2
-	if err := srvtestutils.PostStatusJSONProto(ts, "v2/hotranges", &serverpb.HotRangesRequest{}, &hotRangesResp); err != nil {
-		t.Fatal(err)
-	}
-
-	if len(hotRangesResp.Ranges) == 0 {
-		t.Fatalf("didn't get hot range responses from any nodes")
-	}
-	lastQPS := math.MaxFloat64
-	for _, r := range hotRangesResp.Ranges {
-		if r.QPS > 0 {
-			if r.ReadsPerSecond == 0 && r.WritesPerSecond == 0 && r.ReadBytesPerSecond == 0 && r.WriteBytesPerSecond == 0 {
-				t.Errorf("qps %.2f > 0, expected either reads=%.2f, writes=%.2f, readBytes=%.2f or writeBytes=%.2f to be non-zero",
-					r.QPS, r.ReadsPerSecond, r.WritesPerSecond, r.ReadBytesPerSecond, r.WriteBytesPerSecond)
-			}
-			// If the architecture doesn't support sampling CPU, it
-			// will also be zero.
-			if grunning.Supported && r.CPUTimePerSecond == 0 {
-				t.Errorf("qps %.2f > 0, expected cpu=%.2f to be non-zero",
-					r.QPS, r.CPUTimePerSecond)
-			}
-		}
-		if r.QPS > lastQPS {
-			t.Errorf("unexpected increase in qps between ranges; prev=%.2f, current=%.2f",
-				lastQPS, r.QPS)
-		}
-		lastQPS = r.QPS
-	}
-
-}
-
 func TestHotRanges2Response(t *testing.T) {
```

Comment:
Revised, I see the test for the v2 api here, this looks good - thanks for the cleanup.

Reply:
Thank you @angles-n-daemons